ga4gh/datamodel/reads.py

```diff
@@ -68,6 +68,8 @@ class SamFlags(object):
     """
     NUMBER_READS = 0x1
     PROPER_PLACEMENT = 0x2
+    UNMAPPED = 0x4
+    NEXT_SEGMENT_UNMAPPED = 0x8
```

Comment:
Here's a question: is there a way in the GA4GH schema for a ReadAlignment object to say that its read mate is present, but unmapped?  The `nextMatePosition` attribute seems to be the most likely candidate, but I take a null on that to mean that the read mate is not present.  If there is not a way in the schema to distinguish these two states from each other, then we need to file a schema issue since there's no way to fix the failing test at present.